src/utils/image-handler.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const fallback = require('mdast-util-to-hast/lib/handlers').image;
+
+function image() {
+  return function handler(h, node) {
+    // rewrite blob links to use Fastly
+    if (/^https:\/\/hlx\.blob\.core\.windows\.net\/external\//.test(node.url)) {
+      const patchednode = {
+        ...node,
+      };
+      patchednode.url = `${node.url.replace(/^https:\/\/hlx\.blob\.core\.windows\.net\/external\//, '/hlx_')}.jpg`;
```

Comment:
I find it sub-optimal, that we lose the extension. we could convey it by using a `#hash`, eg:

```
https://hlx.blob.core.windows.net/external/b7a97d3fc2c553bacf17a6e0d69d3e6e7fbab63b#image.png
```

@trieloff WDYT?

Reply:
Yes, that would be easy to do.